### DISCUSSION TITLE
Syhix's Home Page

### DISCUSSION THREAD

**Discussion by Syhix** [Syhix/Syhix.tv#3] [General]
Hello everyone

**Comment by Syhix** [comment]
# **Work in progress**

![](https://cdn.discordapp.com/emojis/442252612956454912.gif?size=96&quality=lossless)

**Comment by Syhix** [comment]
test

**Comment by Hitokii** [comment]
Pas mal ce site en construction __*:kek:*__

**Reply by Syhix** [reply]
![](https://cdn.discordapp.com/emojis/798171533733658685.webp?size=96&quality=lossless)

**Reply by Hitokii** [reply]
![kek](https://i.imgur.com/y25JFtw.jpeg)

**Comment by MDCYT** [comment]
Hi

**Reply by Syhix** [reply]
hi

**Comment by Hitokii** [comment]
POV : j'attend une update du site.
![Dead skeletton meme](https://i.kym-cdn.com/photos/images/facebook/000/643/028/442.jpg)

**Comment by Onelsee** [comment]
Wassup !
